src/compiler/compile/compiler_warnings.ts

```diff
@@ -143,5 +143,9 @@ export default {
 	redundant_event_modifier_passive: {
 		code: 'redundant-event-modifier',
 		message: 'The passive modifier only works with wheel and touch events'
+	},
+	avoid_mouse_events_on_document: {
+		code: 'avoid-mouse-events-on-document',
+		message: 'Mouse events on the document are not supported cross-browser and should be avoided'
```

Comment:
the wording for this error message sounds a bit funny to me. how about?
```suggestion
		message: 'Mouse events on the document are not supported in all browsers and should be avoided'
```